Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8
```

Comment:
Openjdk is pretty large, this definitely lends itself well to a multi stage dockerfile